ext/ACE1pack_Plots_ext.jl

```diff
@@ -0,0 +1,13 @@
+
+module ACE1pack_Plots_ext 
```

Comment:
@cortner, Can you make a final decision about this before we merge?

Reply:
I will remove this.

Reply:
done